lisp/cl-cffi-gtk/Portfile

```diff
@@ -21,6 +21,7 @@ description             A Lisp binding to GTK 3
 long_description        {*}${description}
 
 # test system requires X11, maybe use Xvfb one day?
+test.run                no
```

Comment:
why is this needed; MacPorts' default is set like this, unless you have it enabled by default in the lisp PG?

Reply:
It is enabled in lisp PG because almost all ports have working tests :)